README.md

```diff
@@ -1,13 +1,36 @@
 # Chainblocks examples
 
-Example games and tutorials using [chainblocks](https://github.com/fragcolor-xyz/chainblocks).
+[![license](https://img.shields.io/github/license/fragcolor-xyz/chainblocks-examples)](./LICENSE)
+[![docs](https://img.shields.io/badge/docs-API-blueviolet)](https://docs.fragcolor.xyz/)
 
-## Tutorials
+This repository is a collection of example games and tutorials using the [Chainblocks](https://github.com/fragcolor-xyz/chainblocks) programming language.
+
+Chainblocks is a lisp and hence is extremely powerful as well as fun to learn.
```

Comment:
lisp inspired etc I'd say. it's not a lisp per se

Reply:
Fixed